pkg/datapath/linux/node.go

```diff
@@ -1446,6 +1470,101 @@ func (n *linuxNodeHandler) NodeNeighborRefresh(ctx context.Context, nodeToRefres
 	}
 }
 
+// NodeCleanNeighbors cleans all neighbor entries for previously used network
```

Comment:
s/for previously used network link interfaces/of previously used neighbor discovery link interface/